.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "utils/libphonenumber"]
```

Comment:
If I'm not mistaken, we only use the submodule for one file: `/libphonenumber/resources/PhoneNumberMetadata.xml`

Since submodules are often messy with Git, I'd rather updae the documentation of the python script and say "you need to download the file at https://github.com/googlei18n/libphonenumber/blob/master/resources/PhoneNumberMetadata.xml" or maybe  even download it directly from raw.github.com

That way, we don't need the submodule at all.